pkg/tcpip/tcpip.go

```diff
@@ -1044,10 +1045,24 @@ func isZeros(a Address) bool {
 
 // LinkAddress is a byte slice cast as a string that represents a link address.
 // It is typically a 6-byte MAC address.
-type LinkAddress string
+type LinkAddress uint64
```

Comment:
What about making LinkAddress an alias of [8]byte instead of uint64? We have talked about moving the other address types to fixed length arrays, but haven't gotten around to it yet.

Reply:
NOTE: This assumes that the underlying link is always an ethernet address. What is the real advantage in doing this?